src/main/java/com/dataguise/saas/dto/SourcesDetails.java

```diff
@@ -0,0 +1,97 @@
+package com.dataguise.saas.dto;
+
+import java.sql.Timestamp;
+import java.util.List;
+
+public class SourcesDetails {
+    private int sourceId;
+    private String sourceName;
+    private String sourceType;
+    private String sourceModule;
+    private String sourceAgent;
+    private String sourceHostName;
+    private Integer sourcePortNumber;
+    private List<String> sourceDatabases;
+    private List<String> sourceSchemas;
+
+    public Timestamp getCreatedAt() {
+        return createdAt;
+    }
+
+    public void setCreatedAt(Timestamp createdAt) {
+        this.createdAt = createdAt;
+    }
+
+    private Timestamp createdAt;
+    public String getSourceHostName() {
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocMethodCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/dto/SourcesDetails.java L26
    
    Missing a Javadoc comment.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1009404547>*